src/mockCats.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+const cats = [
+    {
+      id: 1,
+      name: "Remii",
+      age: 1,
+      enjoys: "Eating and throwing her bowl",
+      image:
+        "https://unsplash.com/photos/white-and-brown-long-fur-cat-ZCHj_2lJP00"
+    },
+    {
+      id: 2,
+      name: "Mako",
+      age: 7,
+      enjoys: "being a menace to Andre",
+      image:
+        "https://www.istockphoto.com/photo/stern-sad-cat-isolated-on-white-background-gm1434414228-476036446?utm_campaign=srp_photos_top&utm_content=https%3A%2F%2Funsplash.com%2Fs%2Fphotos%2Fcats&utm_medium=affiliate&utm_source=unsplash&utm_term=cats%3A%3A%3A"
+    },
+    {
+      id: 3,
+      name: "Nala",
+      age: 3,
+      enjoys: "likes to be carried like a baby",
+      image:
+        "https://www.istockphoto.com/photo/cute-ginger-cat-gm1443562748-482502032?utm_campaign=srp_photos_top&utm_content=https%3A%2F%2Funsplash.com%2Fs%2Fphotos%2Fcats&utm_medium=affiliate&utm_source=unsplash&utm_term=cats%3A%3A%3A"
+    }
```

Comment:
Having the value for the images on a separate line might pose an issue when you render these images to the screen.